app/code/community/Jowens/JobQueue/Model/Worker.php

```diff
@@ -53,7 +53,7 @@ public function executeJobs($schedule=null) {
             $collection->addFieldToFilter('queue', array('eq' => $this->getQueue()))
             ->addFieldToFilter('run_at', array(
                 array('null' => true),
-                array('lteq'=> date('Y-m-d H:i:s', Mage::app()->getLocale()->storeTimeStamp()))                 
+                array('lteq'=> date('Y-m-d H:i:s'))

```

Comment:
Nice find here. I'm thinking something like below might make it more obvious that a UTC/GMT date is being used. Can you test it out and make sure it works? 
```
array('lteq' => Mage::getSingleton('core/date')->gmtDate())
```